src/convertToObject.js

```diff
@@ -3,16 +3,35 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let separateElements = sourceString.split(';');
+
+  separateElements = separateElements.map(elem =>
+    (elem.split(':').map(x => x.trim())));
+
+  const result = {};
+
+  separateElements.map(elem => {
+    if (elem.length > 1) {
+      createObjectProp(elem[0], elem[1], result);
+    }
+  });
+
+  function createObjectProp(key, value, obj) {
+    obj[key] = value;
+  }
```

Comment:
it's nice you're introducing helper function, yet it's too short to be honest
i'd rather do it inside the loop right away - it would be simply shorter
also better de-structure elem instead of using `elem[0] / elem[1]` - it will be more explanatory
```suggestion
  separateElements.map(([cssKey, cssValue]) => {
    if (elem.length > 1) {
      result[cssKey] = cssValue;
    }
  });
```

Reply:
Ah yeah its true its too short to create a function for it, i love the helper functions maybe too much :D I fixed it, thanks for the suggestion! :)